lib/migration/plugins/init.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugins
```

Comment:
Please, do not operate by term "plugin" this way. Plugin is used to be something that is dynamically loaded.
Also, Go1.8 will have a real plugin subsystem, which may lead to heavy confusion.